components/ai_chat/core/browser/ai_chat_service_unittest.cc

```diff
@@ -446,11 +447,13 @@ TEST_P(AIChatServiceUnitTest,
 
   // Function to call to finish generating the response.
   base::OnceClosure resolve;
-  EXPECT_CALL(*engine, GenerateAssistantResponse(_, _, _, _, _, _))
+  EXPECT_CALL(*engine, GenerateAssistantResponse)
       .WillOnce(
-          [&resolve](bool send_page_contents, const std::string& page_contents,
+          [&resolve](const bool& is_video, const std::string& page_contents,
```

Comment:
use just bool, bool is one byte and reference is bigger than that

Reply:
Seems sensible, but I was just making it consistent with the actual class implementation which is `const bool& is_video` so seems like we should change it in `EngineConsumer`. OK to keep here for consistency?

Reply:
yeah, I just realized that is out of scope of this PR. Maybe you can fix it in other PRs.